### DISCUSSION TITLE
Help with Updating Server Pro

### DISCUSSION THREAD

**Discussion by RichardCVTC** [rustdesk/rustdesk-server-pro#268] [Q&A] [answered]
Hello I am very new to I.T work. i with the help of our system administrator have set up Rust Desk on hyper v and it has been running great since Jan 2024. i am have trouble updating the server, I run the docker pull command when ever there is an update but i am still on 1.2.0. 
any help would be greatly appreciated.

**Comment by rustdesk** [comment, marked as answer]
https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/faq/#there-is-a-new-version-of-rustdesk-server-pro-out-how-can-i-upgrade
